cosmotheka/mappers/mapper_PR4CMBK.py

```diff
@@ -0,0 +1,43 @@
+from .mapper_P18CMBK import MapperP18CMBK
+import numpy as np
+
+
+class MapperPR4CMBK(MapperP18CMBK):
+    """
+    Note that this mapper is a child of `MapperBase`, /
+    not of `MapperPlanckBase`.
+
+    **Config**
+
+        - file_klm: `".../Datasets/Planck_lensing/Lensing2018/MV/dat_klm.fits"`
+        - file_mask: `".../Datasets/Planck_lensing/Lensing2018/mask.fits.gz"`
+        - file_noise: `".../Datasets/Planck_lensing/Lensing2018/MV/nlkk.dat"`
+        - mask_aposize: `0.2`
+        - mask_apotype: `"C12"`
+        - mask_name: `"mask_P18kappa"`
+        - path_rerun: `".../Datasets/Planck_lensing/Lensing2018/xcell_runs"`
+
+    """
+    map_name = 'PR4CMBK'
+
+    def get_cl_fiducial(self):
+        raise NotImplementedError("Fiducial signal C_ell not provided for "
+                                  "PR4 CMB lensing map.")
+
+    def _get_noise(self):
+        # Returns the decoupled noise power spectrum of the \
+        # auto-correlation of the covergence map.
+
+        # Returns:
+        #     [l (Array): multipole list,
+        #      Nl (Array): noise power spectrum] (Array)
+
+        if self.noise is None:
+            # Read noise file. Column order is: ['l', 'Nl'], 'Nl+Cl']
+            nl = np.loadtxt(self.config['file_noise'])
+            # Zero nan
+            nl[0] = 0
+            ls = np.arange(len(nl))
+            self.noise = (ls, nl)
```

Comment:
This is not consistent with `MapperP18CMBK` definition of noise https://github.com/Cosmotheka/Cosmotheka/blob/2ad07fa06e06341c0e411e5652949219efc6de84/cosmotheka/mappers/mapper_P18CMBK.py#L116

Why do you implement a new class instead of modifying the parent class one? I'd do as you did above, and include this nl[0] = 0 in `MapperP18CMBK` method.

Reply:
The format of the noise file is completely different for PR4 (enough that writing a new function sounds like a better idea to me)

Reply:
hmm, then, is the comment in L.36 correct? It looks pretty much the same as for `MapperP18CMBK`

Reply:
you're right, the comment is wrong. I've fixed it.